packages/core/src/system/dconf.coffee.md

```diff
@@ -0,0 +1,35 @@
+
+# `nikita.system.dconf`
+
+dconf is a low-level configuration system and settings management used by
+Gnome. It is a replacemet of gconf, replacing its XML based database with a
+BLOB based database.
+
+## Options
+
+*   `properties` (object)
+    Name of the module.tools
+
+## Example
+```javascript
+require('nikita').system.dconf({
+  properties: {
+    '/org/gnome/gnome-session/auto-save-session': 'true'
```

Comment:
Unless I'm mistaken, this option caused my only crash with Arch Linux, could you use another property such as "/org/gnome/desktop/datetime/automatic-timezone" or even more (boolean, string, array) with "/org/gnome/desktop/input-sources/xkb-options" and "/org/gnome/desktop/peripherals/touchpad/click-method" as well.

Reply:
You can find some example in [nikita-arch](https://github.com/adaltas/node-nikita-arch/blob/master/lib/system/2.dev_apps.coffee#L23-L56)